### DISCUSSION TITLE
howto build executable so that it has vrl subcommand

### DISCUSSION THREAD

**Discussion by tshepang** [vectordotdev/vector#13335] [Q&A] [answered]
`cargo build` does not work on latest (754d52bf0cf713c975676dbae2f017654a795a95)

```
❯ cargo run --release -- vrl
    Finished release [optimized] target(s) in 0.30s
     Running `target/release/vector vrl`
error: Found argument 'vrl' which wasn't expected, or isn't valid in this context
```

**Comment by jszwedko** [comment, marked as answer]
Hi @tshepang !

Thanks for reporting that, it looks like `vector vrl` was accidentally dropped from the default compilation features. I'll fix that.

As a side note, I'd recommend running `cargo run` from the `lib/vrl/cli` directory to avoid building all of Vector just to run the VRL CLI which should cut down on compilation time.

**Comment by tshepang** [comment]
fixed by #13347
